esx_service/vmdk_ops.py

```diff
@@ -104,9 +104,9 @@
 MAX_SKIP_COUNT = 16       # max retries on VMCI Get Ops failures
 VMDK_ADAPTER_TYPE = 'busLogic'  # default adapter type
 
-#VMCI Version
-# !!! server side VMCI version, need to change according with client side version !!! 
-VMCI_VERSION = "12"
+# Server side understand protocol version. If you are changing client/server protocol we use
```

Comment:
"VMCI_VERSION" has been changed to "PROTOCOL_VERSION". I think both the client and server side use the "PROTOCOL_VERSION" is clear enough.